riff-raff/app/deployment/actors/DeployGroupRunner.scala

```diff
@@ -221,6 +222,22 @@ class DeployGroupRunner(
       )
   }
 
+  private def warnAboutNonStandardProdDeployment(): Unit = {
+    val widelyUsedDefaultBranchNames = List("main", "master")
+    record.metaData.get("branch").foreach { branchName =>
+      if (
+        !widelyUsedDefaultBranchNames.contains(
+          branchName
+        ) && record.stage.name.startsWith("PROD")
+      ) {
+        rootReporter.warning(
+          s"A branch ($branchName) other than main was deployed to PROD. This is a high-risk operation that is strongly discouraged." +
+            "\nSee https://riffraff.gutools.co.uk/docs/riffraff/prod-feature-branches.md for more details."
```

Comment:
This link always points at `PROD` (I didn't think it was worth the added complexity of making it stage-aware), so it won't work until this PR is merged.